.travis.yml

```diff
@@ -92,6 +93,7 @@ script:
     pip install -e .
     # relevant yt version
     pushd $YT_DIR
+    cat .git/HEAD
```

Comment:
Can probably remove this.

Reply:
Good call.  I added that just to check in the travis log that what I thought was happening was actually happening.